Code/app/src/main/java/com/cmpt362/fitcheck/firebase/User.kt

```diff
@@ -0,0 +1,3 @@
+package com.cmpt362.fitcheck.firebase
+
+data class User(var email: String = "", var uid: String = "") {}
```

Comment:
Actually I was reading this page: https://firebase.google.com/docs/database/android/read-and-write

And if you scroll down to their user example, it is similar to what you have. I think what you did before was correct, because of the same reasoning that is mentioned on that page in the example.